CMakeLists.txt

```diff
@@ -160,6 +160,7 @@ if("$ENV{VERBOSE}" STREQUAL "1")
   message(STATUS " Verbose Makefile ACTIVATED")
   set(CMAKE_VERBOSE_MAKEFILE ON)
 endif()
+set(CMAKE_VERBOSE_MAKEFILE ON)
```

Comment:
Please remove this. Instead you should specify `VERBOSE=1` when running `make`, or `-v` when running ninja.

Reply:
sure, this is just debugging the windows build issues on GPU